platform/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: The Particular Service Platform
-reviewed: 2018-12-31
+reviewed: 2019-06-07
+suppressRelated: true
```

Comment:
What does this do?

Reply:
It suppresses the default list of "Related Articles" formed by directory traversal that makes up the majority of https://docs.particular.net/nservicebus/ right now. On a page like this, that's what the menu is for.